models.rdf

```diff
@@ -0,0 +1,93 @@
+      <rdfs:label xml:lang="en">Collection</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        A Collection is a group of resources. Collections have descriptive metadata, access metadata,
+        and may links to works and/or collections. By default, member works and collections are an
+        unordered set, but can be ordered using the ORE Proxy class.
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdfs:Class>
+
+    <rdfs:Class rdf:about="http://pcdm.org/models#Object">
+      <rdfs:label xml:lang="en">Object</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        An Object is an intellectual entity, sometimes called a "work", "digital object", etc.
+        Objects have descriptive metadata, access metadata, may contain files and other Objects as
+        member "components". Each level of a work is therefore represented by an Object instance,
+        and is capable of standing on its own, being linked to from Collections and other Objects.
+        Member Objects can be ordered using the ORE Proxy class.
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdfs:Class>
+   
+    <rdfs:Class rdf:about="http://pcdm.org/models#File">
+      <rdfs:label xml:lang="en">File</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        A File is a sequence of binary data and is described by some accompanying metadata.
+        The metadata typically includes at least basic technical metadata (size, content type,
+        modification date, etc.), but can also include properties related to preservation,
+        digitization process, provenance, etc. Files MUST be contained by exactly one Object.
+      </rdfs:comment>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdfs:Class>
+
+    <rdf:Property rdf:about="http://pcdm.org/models#hasFile">
+      <rdfs:label xml:lang="en">has file</rdfs:label>
+      <rdfs:comment xml:lang="en">Links to a File contained by this Object.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://pcdm.org/models#Object"/>
+      <rdfs:range rdf:resource="http://pcdm.org/models#File"/>
+      <rdfs:subPropertyOf rdf:resource="http://www.w3.org/ns/ldp#contains"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdf:Property>
+
+    <rdf:Property rdf:about="http://pcdm.org/models#hasMember">
+      <rdfs:label xml:lang="en">has member</rdfs:label>
+      <rdfs:comment xml:lang="en">Links to a related Object. Typically used to link to component parts, such as a book linking to a page.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:range rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
```

Comment:
This is precisely what confused me in the [documentation](https://wiki.duraspace.org/display/FF/Portland+Common+Data+Model). The `hasMember` property applies to both `pcdm:Object` and `pcdm:Collection`. One option would be to create a `pcdm:Resource` that subclasses the `ore:Aggregation` class, from which both `pcdm:Object` and `pcdm:Collection` are derived. That class could then be used as a domain and range, though it doesn't address the constraint that a `pcdm:Collection` cannot be a member of a `pcdm:Object`.